### ISSUE TITLE
Add SAM Resource Types to CloudFormation Templates

### ISSUE THREAD

**Issue by byumark** [localstack/localstack#762]
If these 3 resource types were available in cloudformation templates, that would be awesome:
```
AWS::Serverless::Function
AWS::Serverless::Api
AWS::Serverless::SimpleTable
```
The cloudformation service would also have to support the `Transform` section.
```
Transform: 'AWS::Serverless-2016-10-31'
```

**Comment by bnegrao** [comment]
+1

**Comment by abelperezok** [comment]
+1

**Comment by natemacinnes-okta** [comment]
+1

**Comment by goldsam** [comment]
SAM support is kind of important.Can this get some love any time soon?

**Comment by whummer** [comment]
Thanks for reporting @byumark @bnegrao @abelperezok @natemacinnes-okta @goldsam. This should be fixed in #1766 . Closing this issue - please re-open if the problem persists.
